applications/NXmx.nxdl.xml

```diff
@@ -816,6 +822,27 @@
 						</doc>
 					</field>
 
+					<field name="flux_integrated" type="NX_FLOAT" units="NX_FLUX" minOccurs="0">
+						<doc>
+							Flux density incident on beam plane area in photons
+							per per unit area. Useful where temporal profiles of flux
+							are not known
+
+							In the case of a beam that varies in flux shot-to-shot,
+							this is an array of values, one for each recorded shot.
+						</doc>
+					</field>
+
+					<field name="total_flux_integrated" type="NX_FLOAT" units="NX_FREQUENCY" minOccurs="0">
```

Comment:
Units: NX_DIMENSIONLESS

Reply:
Merge https://github.com/nexusformat/definitions/pull/1082 to fix.